reference/3.0/Microsoft.PowerShell.Core/About/about_pipelines.md

```diff
@@ -180,86 +141,149 @@ Get-Process winlogon | Format-List -Property *
+```
+
+or
 
-When you "pipe" objects, that is send the objects in the output of one command
-to another command, PowerShell tries to associate the piped objects with one
-of the parameters of the receiving cmdlet.
+```powershell
+Get-Help Start-Service -Parameter *
+```
 
-To do so, the PowerShell "parameter binding" component, which associates input
-objects with cmdlet parameters, tries to find a parameter that meets the
-following criteria:
+The help for the `Start-Service` cmdlet shows that only the **InputObject** and
+**Name** parameters accept pipeline input.
 
-- The parameter must accept input from a pipeline (not all do).
-- The parameter must accept the type of object being sent or a type that the
-  object can be converted to.
-- The parameter must not already be used in the command.
+```Output
+-InputObject <ServiceController[]>
+Specifies ServiceController objects representing the services to be started.
+Enter a variable that contains the objects, or type a command or expression
+that gets the objects.
+
+Required?                    true
+Position?                    0
+Default value                None
+Accept pipeline input?       True (ByValue)
+Accept wildcard characters?  false
+
+-Name <String[]>
+Specifies the service names for the service to be started.
+
+The parameter name is optional. You can use Name or its alias, ServiceName,
+or you can omit the parameter name.
+
+Required?                    true
+Position?                    0
+Default value                None
+Accept pipeline input?       True (ByPropertyName, ByValue)
+Accept wildcard characters?  false
+```
+
+When you send objects through the pipeline to `Start-Service`, PowerShell
+attempts to associate the objects with the **InputObject** and **Name**
+parameters.
+
+### Methods of accepting pipeline input
+
+Cmdlets parameters can accept pipeline input in one of two different ways:
+
+- **ByValue**: The parameter accepts values that match the expected .NET type
+  or that can be converted to that type.
+
+  For example, the **Name** parameter of `Start-Service` accepts pipeline input
+  by value. It can accept string objects or objects that can be converted to
+  strings.
+
+- **ByPropertyName**: The parameter accepts input only when the input object
+  has a property of the same name as the parameter.
+
+  For example, the Name parameter of `Start-Service` can accept objects that
+  have a **Name** property. To list the properties of an object, pipe it to
+  `Get-Member`.
+
+Some parameters can accept objects by both value or property name, making it
+easier to take input from the pipeline.
+
+### Parameter binding
+
+When you pipe objects from one command to another command, PowerShell attempts
+to associate the piped objects with a parameter of the receiving cmdlet.
+
+PowerShell's parameter binding component associates the input objects with
+cmdlet parameters according to the following criteria:
+
+- The parameter must accept input from a pipeline
```

Comment:
Add periods to the list items.

Change was not to wasn't

Reply:
fixed